tests/integration/base.py

```diff
@@ -150,15 +151,40 @@ def databricks_cluster_profile(self):
             },
             'test': {
                 'outputs': {
-                    'dbsql': {
+                    'cluster': {
                         'type': 'databricks',
                         'host': os.getenv('DBT_DATABRICKS_HOST_NAME'),
                         'http_path': os.getenv('DBT_DATABRICKS_CLUSTER_HTTP_PATH'),
                         'token': os.getenv('DBT_DATABRICKS_TOKEN'),
                         'schema': self.unique_schema()
                     },
                 },
-                'target': 'dbsql'
+                'target': 'cluster'
+            }
+        }
+
+    def databricks_uc_cluster_profile(self):
+        return {
+            'config': {
+                'send_anonymous_usage_stats': False
+            },
+            'test': {
+                'outputs': {
+                    'uc_cluster': {
+                        'type': 'databricks',
+                        'host': os.getenv('DBT_DATABRICKS_HOST_NAME'),
+                        'http_path': os.getenv('DBT_DATABRICKS_UC_CLUSTER_HTTP_PATH'),
+                        'token': os.getenv('DBT_DATABRICKS_TOKEN'),
+                        'schema': self.unique_schema(),
+                        # TODO: catalog should be set as 'catalog' or 'database'
+                        #       instead of using 'session_properties'
+                        # 'catalog': os.getenv('DBT_DATABRICKS_UC_DEFAULT_CATALOG', 'main'),
+                        'session_properties': {
+                            'databricks.catalog': os.getenv('DBT_DATABRICKS_UC_DEFAULT_CATALOG', 'main'),
```

Comment:
We can change the default catalog by changing this session config? I thought we can only 1) set the static config for the initial catalog name before cluster startup or  2) use `USE CATALOG main` before running the query.

Also just curious is there any function we can call to validate cluster configuration before the integration test?  For example for UC profiles, we must have `SET spark.databricks.unityCatalog.enabled` returns `true`.

Reply:
> We can change the default catalog by changing this session config? I thought we can only 1) set the static config for the initial catalog name before cluster startup or  2) use `USE CATALOG [catalog_name]` before running the query.

Yes, `databricks.catalog` in `session_properties` will change the current catalog similar to `USE CATALOG [catalog_name]`

> Also just curious is there any function we can call to validate cluster configuration before the integration test?  For example for UC profiles, we must have `SET spark.databricks.unityCatalog.enabled` returns `true`.

That's a good point. I'll add the tests.

Reply:
Added a test for `databricks_uc_cluster` profile: https://github.com/databricks/dbt-databricks/blob/d500f7c28409836f21b8e3a0fd40bd309fdce82f/tests/integration/current_catalog/test_current_catalog.py#L13-L19
The `databricks_uc_sql_endpoint` profile will raise an error because the UC SQL Endpoint doesn't have such a config.

Another test in the file is checking the current catalog, btw.